### PR TITLE
Add submodules to repo

### PR DESCRIPTION
**Pull request by ppwadhwa**
Replaces examples in nebari with submodules that point to project repos with better examples for each of the projects.

### CODE REVIEW

**Comment by pavithraes** [comment]
@ppwadhwa Thank you for this PR, it looks good! 

One question about https://github.com/jkanner/streamlit-dataview -- I see that the project doesn't have a license document, so can we actually use it here?

Also, I'm so sorry about the unintended actions on this PR (assigning, closing, etc.), my mouse was glitching a lot. 😞

**Comment by ppwadhwa** [comment]
@pavithraes That is a good observation. I am unsure.  @kcpevey do you know?

**Comment by kcpevey** [comment]
@pavithraes great question! We probably shouldn't use it 😓. @ppwadhwa can you find us another streamlit example?
Things I'm looking for:
* plots or other eye candy
* interactivity - click here and something happens over there
* app with several different widgets for complexity
* small source data, open source data that its included inside of the git repo - not "run this notebook to get the data"
* a table would be bonus points

**Comment by kcpevey** [comment]
- [ ] change the plotly dash readme to only suggest the opioid example
- [ ] See if you can deploy this streamlit app https://github.com/mwalmsley/galaxy-poster/blob/master/gz_decals_mike_walmsley.py with this environment:

```
- channels:
- conda-forge
dependencies:
- python <=3.10
- cdsdashboards-singleuser>=0.6.3
- cufflinks-py
- dash
- geopandas
- geopy
- geoviews
- gunicorn
- holoviews
- ipykernel
- ipywidgets
- jupyter
- jupyterlab
- jupyter_bokeh
- matplotlib
- nebari-dask
- nodejs
- numpy
- openpyxl
- pandas
- panel
- param
- plotly
- python-graphviz
- rich
- streamlit
- sympy
- voila
- pip
- pip:
  - streamlit-image-comparison
  - noaa-coops
  - dash_core_components
  - dash_html_components
description: null
name: dashboard-v2
prefix: null
```

**Comment by ppwadhwa** [comment]
For the streamlit example, its running on CDS dashboards, but its not showing images to compare.  The app is running--just no images.  I'm trying to set up the environment locally to see if its a problem with the dashboard.

File: dashboard_examples/plotly-dash/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+The code for example apps is found here: https://github.com/plotly/dash-sample-apps
```
Comment (kcpevey) [inline dashboard_examples/plotly-dash/README.md:1]: The first thing we should say here is that this example can be deployed via cds dashboards on nebari using the default `dashboard` environment that ships with nebari. 

Alternately, you can run it individually using the environment.yml file in this folder.

**Comment by ppwadhwa** [comment]
@kcpevey Is there anything else that needs attention here?  As far as I know tell, I addressed the current concerns.

**Review by kcpevey — APPROVED** [review]

**Pull request merged**